src/modules/anual-budget/budget-summary/budget-summary-template.tsx

```diff
@@ -0,0 +1,75 @@
+import { currencyFormatter } from '@utils/formatters'
+import type { BudgetSummaryType } from '@actions/anual-budget/action'
+import BudgetCardDelta from './budget-card-delta'
+import BudgetCardDoughnut from './budget-card-doughnut'
+
+export const BudgetSummary = (
+    summary: BudgetSummaryType,
+    allAcademicUnits?: Boolean
+) => {
+    const stats = [
+        {
+            name: allAcademicUnits
+                ? 'Presupuesto total'
+                : 'Presupuesto de la UA',
+            total: summary?.academicUnitBudgetSummary.value ?? 0,
+            delta: summary?.academicUnitBudgetSummary.delta,
+            date: summary?.academicUnitBudgetSummary.changeDate,
+            indicator: 'number',
+        },
+        {
+            name: 'Consumo Proyectado',
+            total: summary?.projectedBudget ?? 0,
+            of: summary?.academicUnitBudgetSummary.value ?? 0,
+            delta: 2,
+            indicator: 'number',
+        },
+        {
+            name: 'Consumo Ejecutado',
+            total: summary?.spendedBudget ?? 0,
+            of: summary?.projectedBudget ?? 0,
+            delta: summary?.spendedBudget / summary?.projectedBudget ?? 0,
+            indicator: 'graph',
+        },
+    ]
+    return (
+        <div>
+            <dl className="mt-5 grid grid-cols-1 gap-5 md:grid-cols-3">
+                {stats.map((item) => (
+                    <div
+                        key={item.name}
+                        className="overflow-hidden rounded-lg bg-white px-4 py-5 shadow sm:p-6"
+                    >
+                        <dt className="text-base font-normal text-gray-900">
+                            {item.name}
+                        </dt>
+                        <dd className="mt-1 flex items-baseline justify-between md:block lg:flex">
+                            <div className="text-indigo-600 flex items-baseline text-2xl font-semibold">
```

Comment:
Change all black or simil black texts to text-black/70  or less to mantain uniformity with the rest of the UI design. The idea behind this is to not give such a strong contrast and make it more visually appealing.

Screenshot showing the difference, if the text has a lot of contrast, it catches the attention way more than it should be, on top of appearing a bit off since we are going with a soft contrast look.
![image](https://github.com/bojkomatias/uap-vid/assets/50425596/ce78762f-f749-4685-9046-123a77563d71)

Reply:
Updateado mi rey